crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -57,5 +58,9 @@ def compute_resources_available?
       # FIXME: to be implemented
       true
     end
+
+    def ceph_healthy?
```

Comment:
can you move this up to `clusters_healthy?`? and to the question above, i am wondering if this should be `storages_healthy?`